addons/l10n_id_pos/static/src/js/qr_code_popup.js

```diff
@@ -0,0 +1,40 @@
+import {QRPopup} from "@point_of_sale/app/utils/qr_code_popup/qr_code_popup";
+import {patch} from "@web/core/utils/patch";
+import {AlertDialog} from "@web/core/confirmation_dialog/confirmation_dialog";
+import {_t} from "@web/core/l10n/translation";
+
+patch(QRPopup.prototype, {
+    async _confirm() {
+        // Verify whether the payment has been recieved by QRIS
+
+        this.setButtonsDisabled(true);
```

Comment:
Can you check if you can remove all of `setButtonsDisabled` calls in this override? It seems that `execButton` (which calls `_confirm`) properly handles it already.